applications/nrf_desktop/doc/fn_keys.rst

```diff
@@ -1,7 +1,43 @@
 .. _fn_keys:
 
-fn_keys module
+Fn keys module
 ##############
 
-The ``fn_keys`` module replaces the button key ID for special function keys while
-the function key is pressed.
+The ``fn_keys`` module applies Fn key modifier to activate special functions assigned to dual-purpose keys.
+
+Module Events
+*************
+
++----------------+------------------+--------------+------------------+------------------------------------+
+| Source Module  | Input Event      | This Module  | Output Event     | Sink Module                        |
++================+==================+==============+==================+====================================+
+| :ref:`buttons` | ``button_event`` | ``fn_keys``  | ``button_event`` | :ref:`click_detector`              |
+|                |                  |              |                  +------------------------------------+
+|                |                  |              |                  | :ref:`motion` (``motion_buttons``) |
+|                |                  |              |                  +------------------------------------+
+|                |                  |              |                  | :ref:`hid_state`                   |
++----------------+------------------+--------------+------------------+------------------------------------+
+
+Configuration
+*************
+
+The module uses ``button_event`` sent by :ref:`buttons`.
+Make sure mentioned hardware interface is defined.
+
+The module is enabled with ``CONFIG_DESKTOP_FN_KEYS_ENABLE`` option.
+
+You must configure the following options:
+
+* ``CONFIG_DESKTOP_FN_KEYS_SWITCH`` - Fn key button.
+* ``CONFIG_DESKTOP_FN_KEYS_LOCK`` - Fn lock button.
+* ``CONFIG_DESKTOP_STORE_FN_LOCK`` - option defining if device should store Fn lock state after reboot (set by default).
```

Comment:
```suggestion
* ``CONFIG_DESKTOP_STORE_FN_LOCK`` - option for defining if the device should store the Fn lock state after reboot (set by default to storing the state).
```
Set by default to what setting?

Reply:
This option (like many other Kconfig options) may be set (value = y) or unset (value = n).
If the option is set, the device stores Fn lock state after reboot, otherwise it doesn't.
Do you think it requires additional explaination here ?

Reply:
How do I know you are referring to Kconfig options here? "set" can be understood in many ways. This must be clear and not ambiguous.

Regardless, I'm not sure that even something like "(`set` in Kconfig by default)" would help, because the answer to `if the device should store the Fn lock state after reboot` is `yes` or `no`, no `set`/`unset`.

Here are the options:
- `Kconfig option for defining if the device should be set to storing the Fn lock state after reboot (it stores the state by default).`
- `option for defining if the device should store the Fn lock state after reboot (by default, this option is defined as `set` in Kconfig and the device does store the state).`
- `option for defining if the device should store the Fn lock state after reboot (`set` in Kconfig, which is the default setting) or not (then it must be unset in Kconfig).`

Reply:
Options with `CONFIG_` prefix are in general Kconfig options. "bool" Kconfig options may have 2 values: y (yes) or n (no). You can set or unset the options, for example in the Kconfig menu.
Since this is important for all configurations of the modules, I think we may add informations about Kconfig and its usage in project in separate file, for example configuration primer.